src/test/java/org/reaktivity/specification/kafka/internal/KafkaFunctionsTest.java

```diff
@@ -2216,6 +2222,64 @@ public void shouldGenerateProduceDataExtensionWithNullKeyAndNullHeaderValue()
                     Objects.isNull(h.value())));
     }
 
+    @Test
+    public void shouldGenerateKafkaHeaders()
```

Comment:
We need 2 versions of this, one for `merged` and one for `fetch`.
Please follow the naming conventions used by the other methods, such as `shouldGenerateMerged...` and `shouldGenerateFetch...`.